src/main/java/com/projectkorra/core/math/Angle.java

```diff
@@ -0,0 +1,75 @@
+package com.projectkorra.core.math;
```

Comment:
I think it makes more sense to have this new math directory to be a sub-directory of `util` as opposed to the root source directory.